playwright_tests/E2E/support-scenario.test.ts

```diff
@@ -0,0 +1,107 @@
+
+  console.log("Reasonable adjustment");
+  await expect(page.getByRole('heading', { name: 'Reasonable adjustment' })).toBeVisible();
+  await page.getByLabel('I need documents in an').check();
+  await expect(page.getByText('I need adjustments to get to')).toBeVisible();
+  await expect(page.getByText('I need to bring support with')).toBeVisible();
+  await expect(page.getByText('I need something to feel')).toBeVisible();
+  await expect(page.getByText('I need to request a certain')).toBeVisible();
+  await expect(page.getByText('I need help communicating and')).toBeVisible();
+  await expect(page.getByText('Other')).toBeVisible();
+  await page.getByRole('button', { name: 'Next' }).click();
+  await expect(page.getByRole('heading', { name: 'I need documents in an' })).toBeVisible();
+  await expect(page.getByText('Documents in a specified')).toBeVisible();
+  await page.getByLabel('Documents in a specified').check();
+  await page.getByRole('button', { name: 'Next' }).click();
+
+  console.log("Tell us more about the request");
+  await expect(page.getByText('Tell us more about the request')).toBeVisible();
+  await page.getByLabel('Tell us more about the request').click();
+  await page.getByLabel('Tell us more about the request').fill('Test auto comment');
+  await page.getByRole('button', { name: 'Next' }).click();
+  await expect(page.getByText(caseId)).toBeVisible();
+
+  console.log("Review support request");
+  await expect(page.getByRole('heading', { name: 'Review support request' })).toBeVisible();
+  await expect(page.getByRole('cell').getByText('Applicant')).toBeVisible();
+  await expect(page.getByText('Documents in a specified')).toBeVisible();
+  await expect(page.getByText('Test auto comment')).toBeVisible();
+  await expect(page.getByText('Requested')).toBeVisible();
+  await expect(page.getByRole('link', { name: 'Change party name' })).toBeVisible();
+  await expect(page.getByRole('link', { name: 'Change flag type' })).toBeVisible();
+  await expect(page.getByRole('link', { name: 'Change comments' })).toBeVisible();
+  await expect(page.locator('ccd-case-flag-summary-list div').filter({ hasText: 'Status Requested' }).getByRole('definition').nth(1)).toBeVisible();
+
+  console.log("Submit support request");
+  await page.getByRole('button', { name: 'Submit' }).click();
+  await expect(page.locator('cut-alert')).toContainText('has been updated with event: Request support');
+  await expect(page.getByText(caseId+" has been updated")).toBeVisible();
+  await page.getByText('Case flags', { exact: true }).click();
+  await expect(page.getByRole('table', { name: 'Applicant' }).getByRole('caption')).toBeVisible();
+  await expect(page.getByRole('table', { name: 'Respondent' }).getByRole('caption')).toBeVisible();
+  await page.waitForSelector('ccd-case-full-access-view');
+  const ele = await page.$('ccd-notification-banner .govuk-notification-banner__content');
+  expect(ele == null);
```

Comment:
Shouldn't there be an assertion here? expect(ele == null).toBeTruthy()

Reply:
Nice Catch. Thanks